tests/openapi_request.rs

```diff
@@ -0,0 +1,59 @@
+#![feature(map_first_last)]
```

Comment:
I think this line will make the test fail.

Reply:
I made a change : https://github.com/kdy1/rweb/pull/33/commits/282098d5f2d0f13806fc9685525f92863dacc244